app/views/state_file/state_file_pages/about_page.html.erb

```diff
@@ -19,6 +19,13 @@
           <%= t(".section1_html", faq_link: state_faq_path(us_state: 'us')) %>
         </div>
       </div>
+
+      <% unless Flipper.enabled?(:get_your_pdf) %>
+        <div>
+          <p class="text--bold"><%= t(".check_back_for_prior_year_returns") %></p>
+          <%= link_to t(".faq_prior_year_returns"), state_faq_section_path(us_state: :az, section_key: "how_can_i_access_my_2023_state_tax_return") %>
```

Comment:
I'm guessing it doesn't matter that the `us_state` here is `:az` even though this faq should serve folks looking for previous AZ _or_ NY returns?

Reply:
yeah this is just because in prod it's saved under AZ: https://www.fileyourstatetaxes.org/en/az/faq/how_can_i_access_my_2023_state_tax_return